templates/install.rdf

```diff
@@ -7,7 +7,7 @@
     <em:type>2</em:type>
 
     <em:unpack>false</em:unpack>
-    <em:version>1.1</em:version>
+    <em:version><%= version %></em:version>
```

Comment:
So now `package.json#version` is now the only variable that gets interpolated into the install/update RDF files. Makes templating and everything else a lot simpler.